cpp/tests/trajectory/legacy/test_trajectory_derive.cu

```diff
@@ -101,12 +103,12 @@ TEST_F(TrajectoryDerive, BadData)
     gdf_column_view(&bad_timestamp, 0, 0, 0, GDF_TIMESTAMP);
 
     // null pointers
-    CUDF_EXPECT_THROW_MESSAGE(cuspatial::derive_trajectories(bad_x, bad_y,
-                                                             bad_in_id,
-                                                             bad_timestamp,
-                                                             out_id,
-                                                             out_len, 
-                                                             out_offset),
+    CUSPATIAL_EXPECT_THROW_MESSAGE(cuspatial::derive_trajectories(bad_x, bad_y,
```

Comment:
I think just use CUSPATIAL_EXPECT_THROW.

Reply:
Looks like cudf still has `CUDF_EXPECT_THROW_MESSAGE` in all its tests. Is the reason that we don't want to be explicitly testing error messages (i.e. in case we need to localize them later)?

Reply:
Hmmm, maybe I'm misremembering...